src/app/static/src/app/store/filteredData/filteredData.ts

```diff
@@ -58,9 +67,41 @@ export const initialSelectedFilters: SelectedFilters = {
     }
 };
 
+export const initialSelectedChoroplethFilters: SelectedChoroplethFilters = {
+    sex: null,
+    age: null,
+    survey: null,
+    updateByType: function(type: FilterType, value: FilterOption) {
+        switch (type) {
+            case (FilterType.Age):
+                this.age = value;
+                break;
+            case (FilterType.Sex):
+                this.sex = value;
+                break;
+            case (FilterType.Survey):
+                this.survey = value;
+                break;
+        }
+    },
+    getByType: function(type: FilterType): FilterOption | null {
```

Comment:
Where is this used? Also pretty sure store state needs to be serialisable aka no functions.